charts/stable/homer/values.yaml

```
@@ -9,7 +9,7 @@ image:
   # -- image repository
   repository: b4bz/homer
   # -- image tag
-  tag: 21.09.2
+  tag: v22.07.2
```

```suggestion
  # @default -- chart.appVersion
  tag:
```
This makes it so that in the future we only have to update the app version in the Chart.yaml file

---

Whoa. Didn't know that! I will update it soon, thanks. 😁